src/AlertPanel/AlertPanel.tsx

```diff
@@ -11,26 +11,28 @@ import {
 export const AlertPanel = React.forwardRef<HTMLDivElement, AlertPanelProps>(
   (props, ref) => {
     const {
-      title,
-      illustration,
-      icon,
-      warning,
-      error,
-      success,
       bare,
-      small,
       children,
+      error,
+      icon,
+      illustration,
+      primary,
+      small,
+      success,
+      title,
+      warning,
       ...rest
     } = props
 
     return (
       <AlertBannerContainer
+        $bare={bare}
```

Comment:
Why did you change this ?

Reply:
What's the point of having style-related data attributes here?

Reply:
To avoid the creation of class if we change the prop

Reply:
And what's wrong about it?

Reply:
The previous code was not creating classes and was doing the same thing. 
Here it would be alright as we don't really change the props of `AlertPanel` but we shouldn't do this for something else like a button disabled or not. 
So the main reason is to keep the consistency